packages/ffe-tabs/less/tab.less

```diff
@@ -9,57 +9,84 @@
 // Styleguide ffe-tabs.tab
 
 .ffe-tab {
-    background-color: @ffe-grey-cloud;
-    border: solid 2px @ffe-grey-silver;
-    border-bottom: none;
-    color: @ffe-grey-charcoal;
-    cursor: pointer;
-    display: block;
+    color: @ffe-farge-vann;
+    padding: 10px @ffe-spacing-lg;
+    border-radius: 40px;
+    white-space: nowrap;
+    border: 2px solid transparent;
+    background: @ffe-white;
+    margin: 4px;
     font-family: 'SpareBank1-regular', 'MuseoSans-500', arial, sans-serif;
-    font-size: 1rem;
-    outline: none;
-    overflow: hidden;
-    padding: @ffe-spacing-xs @ffe-spacing-lg @ffe-spacing-xs;
-    text-align: center;
-    text-decoration: none;
-    transition: all @ffe-transition-duration @ffe-ease;
-    vertical-align: middle;
     .native & {
         @media (prefers-color-scheme: dark) {
-            background-color: @ffe-grey-charcoal-darkmode;
-            border-color: @ffe-grey-silver-darkmode;
-            color: @ffe-grey-cloud-darkmode;
+            background-color: @ffe-farge-svart;
+            color: @ffe-farge-vann-30;
         }
     }
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: @ffe-blue-royal;
-        background: @ffe-white;
+    @media (max-width: @breakpoint-sm) {
+        padding: 10px 10px;
+    }
+
+    &:hover:not(&--selected) {
+        color: @ffe-white;
+        background: @ffe-farge-vann;
+        .native & {
+            @media (prefers-color-scheme: dark) {
+                background-color: @ffe-farge-vann-70;
+                color: @ffe-farge-svart;
+                &:active {
+                    border-color: @ffe-farge-vann-70;
+                }
+            }
+        }
+    }
+
+    &:focus:not(&--selected) {
+        border: 2px solid @ffe-farge-vann;
+        box-sizing: border-box;
+        border-radius: 22px;
+        box-shadow: 0 0 0 2px @ffe-blue-focus;
```

Comment:
Denne focus-stylingen er lik stylingen fra den gamle visuelle profilen. For konsistensens skyld bør det heller løses på lignende måte som i [de oppdaterte knappene](https://design.sparebank1.no/komponenter/knapper/), med nye farger og litt annen utforming og oppførsel.

Måten det funker på der er at et pseudo-element (`.ffe-button::after`) vises på focus, men kun når man navigerer med tastatur. Relevant styling ligger i [linje 75-106](https://github.com/SpareBank1/designsystem/blob/0b3c37377454f1c1aa9ed019f11474e1df4601b6/packages/ffe-buttons/less/base-button.less#L75) i base-button.less. [Mer info om teknikken på css-tricks.com](https://css-tricks.com/the-focus-visible-trick/).

Reply:
Skjønner! Takk :D